.metamaskrc.dist

```diff
@@ -2,3 +2,5 @@
 PASSWORD=METAMASK PASSWORD
 INFURA_PROJECT_ID=00000000000
 SEGMENT_WRITE_KEY=
+ONBOARDING_V2=1
```

Comment:
Should these default to being enabled? Right now they're disabled by default.

Reply:
in dev mode it will be nice to have these enabled.

Reply:
I would have found it surprising personally. Unless you're working on one of these features, you wouldn't have reason to enable them. We're still maintaining the old UI after all, fixing bugs and making other improvements.

Reply:
Make sense, I removed the value to disable the feature flag